Automattic/sensei
# Enable admins and lesson author to view quiz for lesson with a prerequisite

Fixes #3071 

### Changes proposed in this Pull Request

* With this PR, we give admins and the lesson author the ability to see the lesson quiz without completing the lesson prerequisite. Basically to help teachers and admins in the process of previewing their work (they don't have to pass the prerequisite lesson just to test the current quiz).

### Testing instructions

1. Add a course
2. Add 2 lessons. Make lesson 1 a prerequisite of lesson 2.
3. Add a quiz to lesson 2.
4. View lesson 2 as admin or lesson author. 
5. Even though you have not completed lesson 1, you should see the "View Quiz" button.
